src/tracks/VariantServiceTrack.tsx

```diff
@@ -248,20 +247,21 @@ class VariantServiceTrack extends igv.TrackBase {
                                 }
                             }
 
+                            let fillStyle;
```

Comment:
a matter of opinion / _best practices_ : try not to declare empty variables, if you can avoid it.

consider the alternative:

```javascript
let fillStyle = this.hetvarColor;

if (!call.genotype) {
  fillStyle = this.noGenotypeColor;
} else if (noCall) {
  fillStyle = this.noCallColor;           
} else if (allRef) {
  fillStyle = this.homrefColor;
}
  
```